lib/rubygems/core_ext/kernel_gem.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
-##
+#--
 # RubyGems adds the #gem method to allow activation of specific gem versions
 # and overrides the #require method on Kernel to make gems appear as if they
 # live on the <code>$LOAD_PATH</code>.  See the documentation of these methods
 # for further detail.
+#++
```

Comment:
I think we can completely get rid of the above paragraph, actually. The same information is provided at `#gem` and `#require` documentation, and feels out of context here.